server/get-info-from-googleapis.js

```diff
@@ -72,12 +74,13 @@ const getData = async (rangeName, sheetId) => {
     const sheetData = await getSheets(sheetId);
 
     const team = getSheetValues(sheetData.valueRanges, rangeName);
+    cashedTeam = team;
```

Comment:
@BohdanLeshchyshyn it'll make a bug) because `getData` method uses for fetching two types of data: `careers` and `team`. That's why we will be cache one-time `team` and other-time `careers`)))

You need to do the following:
1) Rename [team](https://github.com/keenethics/keenethics/blob/de65da77e9d6afb5caf88b9bbd0ce015220c6b7b/server/get-info-from-googleapis.js#L76) to `data`. It was my fail)))
2) and place for caching must be [here](https://github.com/keenethics/keenethics/blob/de65da77e9d6afb5caf88b9bbd0ce015220c6b7b/server/get-info-from-googleapis.js#L90) for `careers` and [here](https://github.com/keenethics/keenethics/blob/de65da77e9d6afb5caf88b9bbd0ce015220c6b7b/server/get-info-from-googleapis.js#L89) for `teams`

Reply:
OR you can cache `sheetData` but in this way you need to make bit changes
```js
  } catch (err) {
    console.error(err);
    // TODO: rename cashedTeam to cachedSheetData
    const data = getSheetValues(cashedTeam.valueRanges, rangeName);
    return arrayOfArraysToCollection(data);
  }
```

and this approach better for me